services/backend-for-frontend-nodejs/src/semconv.ts

```diff
@@ -0,0 +1,48 @@
+// It is recommended to use a local copy of constants for experimental semantic conventions to avoid breaking changes.
```

Comment:
This is a file to create for any semantic conventions being used that are not stable. The advice is not to use the incubating entrypoint at all.

Reply:
Delete this for now it's excessive.